2.1.0 forces me to use chromedriver 99

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: nightwatch@2.1.0
npm ERR! Found: chromedriver@100.0.0
npm ERR! node_modules/chromedriver
npm ERR!   dev chromedriver@"^100.0.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peerOptional chromedriver@"^99.0.0" from nightwatch@2.1.0
npm ERR! node_modules/nightwatch
npm ERR!   dev nightwatch@"^2.1.0" from the root project
```

I don't think it should enforce a version...

--- a/package.json
+++ b/package.json
@@ -54,7 +54,7 @@
   },
   "peerDependencies": {
     "@cucumber/cucumber": "^7.3.1",
-    "chromedriver": "^99.0.0",
+    "chromedriver": "*",
     "geckodriver": "*"
   },
   "peerDependenciesMeta": {

